test/membrane_rtmp_plugin/rtmp_source_bin_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule Membrane.RTMP.SourceBin.Test do
```

Comment:
Also the test somehow should be marked as an integration test since we are using the external ffmpeg commands/